src/Altinn.AccessManagement/Controllers/LookupController.cs

```diff
@@ -63,5 +64,45 @@ public async Task<ActionResult<PartyExternal>> GetOrganisation(string orgNummer)
                 return StatusCode(500);
             }
         }
+
+        /// <summary>
+        /// Endpoint for retrieving 
+        /// </summary>
+        /// <param name="partyId">The partyId for the reportee to look up</param>
+        /// <returns>Reportee if party is in authenticated users reporteelist</returns>
+        [HttpGet]
+        [Authorize]
+        [Route("accessmanagement/api/v1/lookup/reportee/{partyId}")]
+        public async Task<ActionResult<PartyExternal>> GetParty(int partyId)
+        {           
+            try
+            {
+                int userId = AuthenticationHelper.GetUserId(HttpContext);
+                List<Party> partyList = await _register.GetPartiesForUser(userId);
+               
+                if (partyList.Count > 0)
+                {
+                    foreach (Party party in partyList)
```

Comment:
Linq is probably little faster here

Reply:
Also, the reporteelist is hierarchical so need to check for child parties. See PartiesWrapper.ValidateSelectedParty in authorization for example: [PartiesWrapper](https://github.com/Altinn/altinn-authorization/blob/main/src/Authorization/Services/Implementation/PartiesWrapper.cs) which does the same but only returns a bool and not the party

Reply:
Moved logit to servicelayer and implemented support for subunits.